### PR TITLE
:dart::pencil: add benchmark files for quarter circle isoparametric 

### PR DESCRIPTION
**Pull request by jgiven100**
**Describe the PR**
* Add benchmark files for 10m and 30m plate with hole model (isoparametric)
* Update `README.md`

**Related Issues/PRs**
https://github.com/cb-geo/mpm-doc/pull/50

**Additional context**
n/a

### CODE REVIEW

**Review by kks32 — CHANGES_REQUESTED** [review]
could you please add results to an ipynb file?

**Comment by jgiven100** [comment]
@kks32 
* added `results.ipynb` for the 10m and 30m models
* files include min and max cartesian stresses at begining, intermediate, and final timesteps for the associated and non-associated analysis
* I was thinking to do radial and hoop stresses instread of cartesian stresses, but that requires a quite a bit more post-processing than what the existing `results.ipynb` seemed to have


Since min and max cartesian stress don't have a particuarly important meaning for this model, please let me know what else to add to these notebooks

File: 2d/plate_with_hole/isoparametric/10m/mpm_nonassociated.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      "xvalues": [0.0, 0.01, 1.0],
+      "fxvalues": [0.0, 1.0, 1.0]
+    }
+  ],
+  "analysis": {
+    "type": "MPMExplicit2D",
+    "stress_update": "usf",
+    "velocity_update": true,
+    "dt": 1.0E-6,
+    "uuid": "testing_nonassociated_",
```
Comment (kks32) [inline 2d/plate_with_hole/isoparametric/10m/mpm_nonassociated.json]: Could we have these as `nonassociated` and `associated`? correspondingly please change ipynb file too.

**Review by kks32 — CHANGES_REQUESTED** [review]

**Comment by kks32** [comment]
@jgiven100 it would be good to add the radial and hoop stresses, we can compute those from the normal stress components.

**Comment by jgiven100** [comment]
> it would be good to add the radial and hoop stresses, we can compute those from the normal stress components.

@kks32 Are you thinking loop through all material points to find max `sigma_rr` and `sigma_tt` or just pick an internal point and show polar stress for reference

**Comment by kks32** [comment]
@jgiven100 just a selected internal point.

**Comment by jgiven100** [comment]
@kks32 
* `testing_` removed from `mpm-input.json` filenames and corresponding `results.ipynb` files
* `results.ipynb` include radial and hoop stress for internal material point 

(sorry that took 3 commits)

**Review by kks32 — APPROVED** [review]

**Comment by kks32** [comment]
Thank you @jgiven100 !

**Pull request merged**